safe/common/test/test_version.py

```diff
@@ -18,9 +18,11 @@
 __copyright__ += 'Disaster Reduction'
 
 
+import os
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->